### PR TITLE
Ganga2022

### PR DESCRIPTION
**Pull request by egede**
A Ganga project for GSoC 2022

### CODE REVIEW

File: _gsocproposals/2022/proposal_GangaMonitoring.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Task ideas
+ * Explore the relative benfits of using different concurrency/parallelism methods (asyncio, threads, multiprocessing)
+ * Set-up a local system that can be used to test and develop the monitoring without having dependencies on
+   externals.
+ * Develop a new monitoring system that utilise concurrency in the optimal way to reduce latency in the system.
+ * Improve the user feedback in Ganga by making it easier to see the progress of the monitoring and to understand what
+   monitoring operations are queued for execution.
+
+## Expected results
+A monitoring system that is able to keep the users session responsive and that doesn't fall behind in the monitoring of tasks.
```
Comment (agheata) [inline _gsocproposals/2022/proposal_GangaMonitoring.md:28]: Saying a bit more about the expected features of the new system would help. Can you please add a sentence here about what the student will learn/achieve in the process? We thought that adding this info to our proposals this year would help make them more attractive.

File: _gsocproposals/2022/proposal_GangaMonitoring.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Evaluation Task
+Interested students please contact Ulrik (see contact below) to ask questions and for an evaluation task.
+
+## Requirements
+Python programming, Experience with concurrency, Linux command line experience
```
Comment (agheata) [inline _gsocproposals/2022/proposal_GangaMonitoring.md:34]: Please add the level of experience required for each category.

**Review by agheata — CHANGES_REQUESTED** [review]

File: _gsocproposals/2022/proposal_GangaMonitoring.md (hunk 1 above)
Comment (egede) [reply _gsocproposals/2022/proposal_GangaMonitoring.md]: Added some more detail

File: _gsocproposals/2022/proposal_GangaMonitoring.md (hunk 2 above)
Comment (egede) [reply _gsocproposals/2022/proposal_GangaMonitoring.md]: Added more detail. Not sure if the wording advanced, intermediate, novice is the right one to use. Comment if not.

**Review by agheata — APPROVED** [review]

**Pull request merged**